kiwrious/src/main/java/org/ahlab/kiwrious/android/KiwriousReader.java

```diff
@@ -173,32 +179,36 @@ public int getB() {
 
     //    ---------------------------------------------------------------------------------------------------------------
 
+    public boolean isSensorOnline(String sensorName){
+        return sensorStatus.get(sensorName);
+    }
+
     public boolean isHumidityOnline() {
```

Comment:
These getters seem a bit redundant now that we have a more generic accessor. I guess they could still be useful.. probably good to clarify in README